site/content/en/docs/tasks/_index.md

```diff
@@ -40,12 +40,12 @@ As a batch user, you can learn how to:
 - [Run a Kueue managed KubeRay RayJob](/docs/tasks/run_rayjobs).
 - [Submit Kueue jobs from Python](/docs/tasks/run_python_jobs).
 - [Run a Kueue managed plain Pod](/docs/tasks/run_plain_pods).
+- [Submit Kueue jobs from Python](/docs/tasks/run_python_jobs).
```

Comment:
This point is already on the list for "Batch user", two positions up.

Reply:
and title the PR accordingly

Reply:
Sure.

Reply:
Done.